afdko/PKGBUILD

```diff
@@ -3,58 +3,63 @@
 
 pkgname=afdko
 pkgver=4.0.1
-pkgrel=1
-pkgdesc='Adobe Font Development Kit for OpenType'
+pkgrel=6
+pkgdesc="Adobe Font Development Kit for OpenType"
 arch=(x86_64)
 url="https://github.com/adobe-type-tools/$pkgname"
-license=(custom)
+license=('Apache-2.0')
 _py_deps=(booleanoperations
           brotli # for fonttools[woff]
           defcon
           fontmath
-          fontparts
           fontpens # for defcon[pens]
           fonttools
           fs # for fonttools[ufo]
-          lxml # for fonttools[lxml] and defcon[lxml]
+          lxml # for defcon[lxml] and fonttools[lxml]
           tqdm
           ufonormalizer
           ufoprocessor
           unicodedata2 # for fonttools[unicode]
           zopfli) # for fonttools[woff]
-depends=(python
-        "${_py_deps[@]/#/python-}"
-        psautohint)
+depends=(gcc-libs
+         glibc
+         libxml2
+         python
+         "${_py_deps[@]/#/python-}")
 makedepends=(cmake
              git # Upstream Issue: https://github.com/adobe-type-tools/afdko/issues/1407
-             python-setuptools-scm
-             python-scikit-build)
-checkdepends=(python-pytest)
+             ninja
+             python-{build,installer,wheel}
+             python-{scikit-build,setuptools-scm})
+checkdepends=(python-{pytest,virtualenv})
 conflicts=(spot-client)
 _archive="$pkgname-$pkgver"
-source=("$url/releases/download/$pkgver/$_archive.tar.gz")
+source=("$_archive.tar.gz::$url/releases/download/$pkgver/$_archive.tar.gz")
```

Comment:
This is unnecessary clutter. You only need to set the output filename if it is *different* from what is being downloaded. In this case it is not.